### DISCUSSION TITLE
How many Olympic Games have been hosted in Africa?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#756] [Galaxy Trivia] [answered]
*answer with a GIF*

**Comment by peterbaccaro** [comment]
0️⃣ - Africa has yet to host an Olympic Games.

**Comment by SANKARAMDAS** [comment]
0️⃣ - Never before has Africa hosted the world's biggest sporting event. However, there is a chance that it will finally take place on the Mother Continent. The Egyptian Ministry of Youth and Sports has announced that it will submit its application to become the organiser of the 2036 Summer Olympics.

**Comment by ParanoidUser** [comment]
*The Graphics Interchange Format (GIF)* is a bitmap image format supports both animated and static images. 🙄

**Reply by peterbaccaro** [reply, marked as answer]
Africa has yet to host an Olympic Games.

![zero](https://github.com/ParanoidUser/yolo/assets/74666079/47942a20-1082-4990-9e08-a9adf32af3fc)
